### PR TITLE
Suse vnfs suse include fixup

### PR DESCRIPTION
**Pull request by jjolly**
wwmkchroot: Fix SUSE specific installation functions to work with openSUSE and SLES
    
    Signed-off-by: Egbert Eich <eich@suse.com>

### CODE REVIEW

File: vnfs/libexec/wwmkchroot/include-suse [hunk 1]
```diff
@@ -1,6 +1,4 @@
-REPO_NAME="SLES12-12-0-sp1"
-ZYP_CONF="/root/yum-ww.conf"
-ZYP_CMD="zypper -c $CHROOTDIR/$ZYP_CONF --root $CHROOTDIR -n --gpg-auto-import-keys"
+ZYP_CMD="zypper --root $CHROOTDIR -n --gpg-auto-import-keys"
```
Comment (bensallen) [inline vnfs/libexec/wwmkchroot/include-suse]: I assume zypper chroots first then loads it config file so specifying the config path isn't needed here?

File: vnfs/libexec/wwmkchroot/include-suse [hunk 2]
```diff
@@ -27,48 +34,27 @@ prechroot() {
...
+	VERSION=`rpm -qf /etc/SuSE-release  --qf '%{VERSION}\n'`
+    fi
     mkdir -p $CHROOTDIR
-    mkdir -p $CHROOTDIR/etc
-    mkdir -p $CHROOTDIR/etc/zypp
-    mkdir -p $CHROOTDIR/dev
+    mkdir -p -m 755 $CHROOTDIR/etc
+    mkdir -p -m 755 $CHROOTDIR/etc/zypp
+    mkdir -p -m 755 $CHROOTDIR/dev
+     rm -f $CHROOTDIR/dev/zero && mknod -m 666 $CHROOTDIR/dev/zero c 1 5
```
Comment (bensallen) [inline vnfs/libexec/wwmkchroot/include-suse]: Extra space of indentation here.

File: vnfs/libexec/wwmkchroot/include-suse [hunk 3]
```diff
@@ -27,48 +34,27 @@ prechroot() {
...
-        cleanup
-        return 1
-    fi
-
-    VERSION=`rpm -qf /etc/SuSE-release  --qf '%{VERSION}\n'`
 
+    if [ -e "/etc/os-release" ]; then
+        VERSION=`rpm -qf /etc/os-release  --qf '%{VERSION}\n'`
+    else
+	VERSION=`rpm -qf /etc/SuSE-release  --qf '%{VERSION}\n'`
```
Comment (bensallen) [inline vnfs/libexec/wwmkchroot/include-suse]: Tab instead of spaces for indentation here.

File: vnfs/libexec/wwmkchroot/include-suse (hunk 1 above)
Comment (jjolly) [reply vnfs/libexec/wwmkchroot/include-suse]: That is a correct assumption. The zypper tool is frequently used on chroot base paths, and we here at SUSE like to minimize the number of options needed in our tools. ;-)

**Comment by jjolly** [comment]
Cleaned up the indentation of the suse-include file. Please review.

**Pull request merged**